tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestDateOrder.java

```diff
@@ -440,18 +451,18 @@ public void testIso8601() {
             },
             {
                 "//ldml/dates/calendars/calendar[@type=\"iso8601\"]/dateTimeFormats/dateTimeFormatLength",
-                "{1} {0}",
-                "Field out of order: {1}…{0}"
+                "{0} {1}",
+                "The {0} field (=time) cannot come before the {1} field (=date), in a YMD calendar."
```

Comment:
```suggestion
                "In a YMD calendar, the {1} field must come before the {0} field."
```

- =time and =date are interesting but possibly superfluous
- changed to say what TO do, instead of emphasizing what went wrong.